nextcord/gateway.py

```diff
@@ -913,27 +952,27 @@ async def initial_connection(self, data):
         _log.info("selected the voice protocol for use (%s)", mode)
 
     @property
-    def latency(self):
+    def latency(self) -> float:
         """:class:`float`: Latency between a HEARTBEAT and its HEARTBEAT_ACK in seconds."""
         heartbeat = self._keep_alive
         return float("inf") if heartbeat is None else heartbeat.latency
 
     @property
-    def average_latency(self):
+    def average_latency(self) -> float:
         """:class:`list`: Average of last 20 HEARTBEAT latencies."""
         heartbeat = self._keep_alive
         if heartbeat is None or not heartbeat.recent_ack_latencies:
             return float("inf")
 
         return sum(heartbeat.recent_ack_latencies) / len(heartbeat.recent_ack_latencies)
 
-    async def load_secret_key(self, data):
+    async def load_secret_key(self, data: Dict[str, Any]) -> None:
         _log.info("received secret key for voice connection")
         self.secret_key = self._connection.secret_key = data.get("secret_key")
         await self.speak()
         await self.speak(SpeakingState.none)
 
-    async def poll_event(self):
+    async def poll_event(self) -> Union[None, NoReturn]:
```

Comment:
```suggestion
    async def poll_event(self) -> None:
```
yes it may raise but `NoReturn` tells type checkers that the code below the call will not run, which is unnecessary here since it may not happen